src/font.rs

```diff
@@ -15,7 +15,7 @@ pub struct Font {
 impl Font {
     /// Creates a new font with the given data and collection index.
     #[must_use]
-    pub fn new(data: Blob<u8>, index: u32) -> Self {
+    pub const fn new(data: Blob<u8>, index: u32) -> Self {
```

Comment:
Hmm. Is it possible to make a Blob in a const context?

Reply:
I'm going to remove this `const` from this PR.